README.md

```diff
@@ -21,13 +21,91 @@ Then generate your new project:
 yo node-graphql
 ```
 
+## Generated code
+
+If all default options are entered, the code generated should include a few runnable tests, as well as a runnable server.
+
+### Tests
+
+The generated tests are written using `Jest`, and provide an in memory database using `mongodb-memory-server`, as well as means to test any graphql resolvers and schema validations using `easygraphql-tester`.
```

Comment:
"using `Jest`, and provide an in memory database using..." => "using `Jest` and provide an in memory database using..."